packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -1514,6 +1537,15 @@ export default class RestApi {
         res.sendStatus(200)
       })
     )
+
+    api.patch(
+      '/groups/update/:id',
+      json(),
+      wrap(async (req, res) => {
+        await collections.groups.actions.update(req.body, req)
+        res.sendStatus(200)
+      })
+    )
```

Comment:
IMHO this can be split into 2 routes.
Update group name:
```
PATCH /:collection(groups)/:id
request body: {name?: string}
```
Add user to group:
```
POST /:collection(groups)/:id/users/:id
```

First route we update a property of the `group` collection
Second we add a user to the `users` subresource of the `group` collection